subcommands/config/updates.go

```diff
@@ -20,11 +20,31 @@ in a device group. When run without options, prints out the current configuratio
   # Make devices start taking updates from Targets tagged with "devel":
   fioctl config updates --group beta --tag devel
 
-  # Set the docker apps that devices will run:
+  # Set the Compose apps that devices will run:
   fioctl config updates --group beta --apps shellhttpd
 
-  # Set the docker apps and the tag for devices:
-  fioctl config updates --group beta --apps shellhttpd --tag master`,
+  # Set the Compose apps and the tag for devices:
+  fioctl config updates --group beta --apps shellhttpd --tag master
+
+	# There are two special characters: "," and "-"
+	# The following explains what happens if you provide a ","
+  # Set the Compose apps to none for devices, this will make the device run no apps:
```

Comment:
```suggestion
  # There are two special characters: "," and "-".
  # Providing a "," sets the Compose apps to "none" for devices.
  # This will make the device run no apps:
```